packages/preload/src/index.ts

```diff
@@ -1619,7 +1619,7 @@ export function initExposure(): void {
 
   contextBridge.exposeInMainWorld(
     'sendShowQuickPickValues',
-    async (quickPickId: number, selectedIndexes: number[]): Promise<void> => {
+    async (quickPickId: number, selectedIndexes: number[] | undefined): Promise<void> => {
```

Comment:
make it optionally like selected indexes?: number []

Reply:
Sure, that's better. Fixed.